src/components/ConfigScreen.css

```diff
@@ -0,0 +1,9 @@
+.ix-helper-text {
+  margin-top: 0.5rem;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial,
+    sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol;
+  font-size: 0.875rem;
+  line-height: 1.5;
```

Comment:
nit: is the the only place where this font styling will be used? My 2c is that be good to scope this out to the app level so you're not constantly having to define `font-family` etc. Something like `.ix-font` or `.ix-text`. Otherwise this lgtm

Reply:
Short answer here is "yes". If you remember, we had to reconstruct the form label because the native one didn't allow for clickable links. This font-family is used everywhere by forma36 but maybe there's a cleaner way to import it from their libraries rather than manually redefine it 🤔